File: cms_web/src/Services/Entities/ConsultVitalSignEntity.ts
interface ConsultVitalSignEntity {
  cr_vs_pk?: number;
  consult_req_pk?: string;
  blood_pressure?: string;
  heart_rate?: string;
  resp_rate?: string;
  temperature?: string;
  height?: string;
  weight?: string;
  bmi?: string;
  remarks?: string;
  is_valid?: "y" | "n";
  encoded_at?: string | Date;
  encoder_pk?: string;
}

export interface ConsultVitalSignTableModel {
  limit: number;
  count: number;
  begin: number;
  table: Array<ConsultVitalSignEntity>;
}

export default ConsultVitalSignEntity;

File: cms_server/cms_server/Entities/ConsultVitalSignEntity.cs
using System;

namespace cms_server.Entities
{
    public class ConsultVitalSignEntity
    {
        public int? cr_vs_pk { get; set; }
        public string consult_req_pk { get; set; }
        public string blood_pressure { get; set; }
        public string heart_rate { get; set; }
        public string resp_rate { get; set; }
        public string temperature { get; set; }
        public string height { get; set; }
        public string weight { get; set; }
        public string bmi { get; set; }
        public string remarks { get; set; }
        public string is_valid { get; set; }
        public DateTime? encoded_at { get; set; }
        public string encoder_pk { get; set; }
    }
}
